test/case/03-test-update-from-master

```diff
@@ -21,43 +21,43 @@ LOG=${0##*/}.log
 boilerplate_master=$(new_boilerplate_repo)
```

Comment:
NOTE TO REVIEWERS: I changed the mode on this file to disable this test case. Because it straddles master and $current_commit, it was going to be harder-than-reasonable to make it work with the renamed convention, and we're going to need to undo whatever accommodation we implement anyway. This was the easiest thing for now.

Reply:
Re-enabled via #30.